packages/components/src/VirtualizedList/utils/resizable.js

```diff
@@ -0,0 +1,267 @@
+import flow from 'lodash/flow';
+
+const MINIMUM_COLUMN_WIDTH = 40;
+
+const getMinimumWidth = column => column.minWidth || MINIMUM_COLUMN_WIDTH;
+
+const getWidth = (width, minWidth = MINIMUM_COLUMN_WIDTH) => (width <= minWidth ? minWidth : width);
+
+/**
+ * Search for the nearest index shrinkable on the right.
+ * @param {integer} index
+ * @param {array} columnsWidths
+ */
+const getShrinkIndexRight = (index, columnsWidths) => {
+	for (let i = index + 1; i < columnsWidths.length; i += 1) {
+		const next = i + 1;
+		// Shrink columns on the right of the current, if resizable and width above minimal.
+		if (columnsWidths[i].resizable && columnsWidths[i].width > getMinimumWidth(columnsWidths[i])) {
+			return i;
+			// Shrink the after first columns on the right, if the next one is already at minimal width.
+		} else if (
+			columnsWidths[i] === getMinimumWidth(columnsWidths[i]) &&
+			next < columnsWidths.length - 1 &&
+			columnsWidths[next].resizable &&
+			columnsWidths[next].width > getMinimumWidth(columnsWidths[next])
+		) {
+			return next;
+		}
+	}
+	return -1;
+};
+
+/**
+ * Search for the nearest index enlargeable on the right.
+ * @param {integer} index
+ * @param {array} columnsWidths
+ */
+const getEnlargeIndexRight = (index, columnsWidths) => {
```

Comment:
Same remark here, don't understand the check by couple with a pace of 1.
It can be simplified
```
const colToResizeIndex = columnsWidths.slice(index).findIndex(colWidth => colWidth.resizable);
return colToResizeIndex === -1 ? -1 : (index + colToResizeIndex);
```

Reply:
I feel stupid... thank you nighty Jimmy